src/Core/Domain/Product/Combination/QueryResult/EditableCombinationForListing.php

```diff
@@ -93,7 +93,7 @@ public function __construct(
         bool $default,
         DecimalNumber $impactOnPrice,
         int $quantity,
-        ?string $imageUrl = null
+        string $imageUrl
```

Comment:
Please document the BC Break as null is not anymore allowed

Reply:
Yep, nice catch